scripts/dummy.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
Yeah, I'm not sure how true that performance difference is still, but since we're going to focus on http requests which are orders of magnitude slower it shouldn't make a difference.
Picking bluebird would be nice for some extra utils if needed